### PR TITLE
Ret 2086 - Post login page

### PR DESCRIPTION
**Pull request by marcjudge**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-2086

### Change description
Adding new post login page which will allow users to view/select existing claim or start a new claim

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/test/unit/views/steps-to-making-your-claim.test.ts [hunk 1]
```diff
@@ -110,12 +110,16 @@ describe('Steps to making your claim page', () => {
           htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
         });
       const links = htmlRes.querySelectorAll(linkClass);
+      for (const key of links) {
+        console.log(key.outerHTML);
```
Comment (lvo700) [inline src/test/unit/views/steps-to-making-your-claim.test.ts:126]: Console log

File: src/test/unit/views/steps-to-making-your-claim.test.ts [hunk 2]
```diff
@@ -221,6 +226,7 @@ describe('Steps to making your claim page', () => {
       'Whistleblowing',
       'Other type of Claim',
     ];
+    console.log(htmlRes);
```
Comment (lvo700) [inline src/test/unit/views/steps-to-making-your-claim.test.ts:241]: Console log

File: src/test/unit/views/steps-to-making-your-claim.test.ts (hunk 1 above)
Comment (marcjudge) [reply src/test/unit/views/steps-to-making-your-claim.test.ts]: Removed

File: src/test/unit/views/steps-to-making-your-claim.test.ts (hunk 2 above)
Comment (marcjudge) [reply src/test/unit/views/steps-to-making-your-claim.test.ts]: Removed

**Review by lvo700 — APPROVED** [review]

**Review by brendanmchugh — APPROVED** [review]

File: src/main/resources/locales/en/translation/common.json [hunk 3]
```diff
@@ -2,6 +2,7 @@
   "startNow": "Start now",
   "back": "Back",
   "continue": "Continue",
+  "view": "view",
```
Comment (DarylBoam) [inline src/main/resources/locales/en/translation/common.json:5]: Should this be "View"?

**Review by DarylBoam — APPROVED** [review]

**Review by JustMehmet — APPROVED** [review]

**Review by DarylBoam — APPROVED** [review]

**Pull request merged**